for_devs/local-net

```diff
@@ -125,7 +125,7 @@ def config(binary, boot_nodes, rpc_nodes):
             cmd += f'--boot-nodes {boot_nodes_flags} '
 
             if type == "boot":
-                cmd += '--boot-node '
+                # cmd += '--boot-node ' // TODO: we should replace this with bootnode binary? 
```

Comment:
we do not need boot nodes on local net I guess

Reply:
But we have to specify count of bootnodes for some reason. So maybe we should remove it option and logic related to this

Reply:
I will create separate task for this